packages/modules/workflow-engine-redis/package.json

```
@@ -29,7 +29,7 @@
     "resolve:aliases": "tsc --showConfig -p tsconfig.json > tsconfig.resolved.json && tsc-alias -p tsconfig.resolved.json && rimraf tsconfig.resolved.json",
     "build": "rimraf dist && tsc --build && npm run resolve:aliases",
     "test": "jest --passWithNoTests --bail --forceExit -- src",
-    "test:integration": "jest  --forceExit -- integration-tests/**/__tests__/*.ts",
+    "test:integration": "jest  --forceExit --runInBand -- integration-tests/**/__tests__/index.spec.ts && jest --forceExit --runInBand -- integration-tests/**/__tests__/race.spec.ts",
```

I guess this should be reverted?

```suggestion
    "test:integration": "jest --forceExit -- integration-tests/**/__tests__/**/*.ts",
```

---

This test has something that running one after the other makes it hang. We could not find yet what is causing this, we just suspect that something is happening on the test runner , or the afterEach/afterAll. So, for now we'll keep it like this, with `&&`

---

Okay, is this something we should look into before the release?

---

No, we are confident this is only affecting this test. They run just fine when executed like this manually one after the other, so it has something to do with the test setup than the code itself